hera_mc/cm_hookup.py

```diff
@@ -188,28 +186,49 @@ def __add_hookup_timing(self, hookup_dict):
                 hookup_dict['timing'][akey][pkey] = [latest_start, earliest_stop]
         return hookup_dict
 
-    def __get_column_header(self, hup0):
-        parts_col = []
-        for hu in hup0:
-            get_part_type = self.handling.get_part_dossier(hpn=hu.upstream_part,
-                                                           rev=hu.up_part_rev,
-                                                           at_date=self.at_date,
-                                                           exact_match=True)
-            pr_key = cm_utils._make_part_key(hu.upstream_part, hu.up_part_rev)
-            parts_col.append(get_part_type[pr_key]['part'].hptype)
-        hu = hup0[-1]
-        get_part_type = self.handling.get_part_dossier(hpn=hu.downstream_part,
-                                                       rev=hu.down_part_rev,
-                                                       at_date=self.at_date,
-                                                       exact_match=True)
-        pr_key = cm_utils._make_part_key(hu.downstream_part, hu.down_part_rev)
-        parts_col.append(get_part_type[pr_key]['part'].hptype)
-        parts_col.append('Start')
-        parts_col.append('Stop')
-        return parts_col
+    def __get_column_headers(self, huh):
+        """
+        This gets the full set of headers for a future show_hookup that doesn't require the same
+        hookup starting point.  Returns a single column for now (the one/long_column stuff)
+        """
+        hu_col = {}
+        return_one_column_for_now = True
+        if return_one_column_for_now:  # This is to just not find part_dossiers on everything for now
```

Comment:
It seems to me like you're trying to future-proof since you know you're going to change this code later. I think it is better to write the code to do what it does right now in the most direct way possible, then change it later when it needs to do something different. You're not going to forget about the need for the future change, unless it turns out that you don't actually need to do it! And it doesn't look like this is a case where there's tricky logic that you've worked out that deserves to preserved so you don't have to spend time recreating it later.